src/bin/tikv-server.rs

```diff
@@ -98,9 +97,13 @@ fn build_raftkv(matches: &Matches,
 }
 
 fn get_store_path(matches: &Matches) -> String {
-    let path = matches.opt_str("s").expect("need store path, but none is specified!");
+    let path = matches.opt_str("s");
+    if path.is_none() {
+        return TEMP_DIR.to_owned();
```

Comment:
The path is different for different engine. We should let the engine to handle path later.